uint16 vector_size
task[] missions
